sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,883 @@
+      "description": "Add additional parameters to the Authorization URL.",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": "spring.cloud.azure.active-directory.authenticate-additional-parameters"
+      }
+    },
+    {
+      "name": "azure.activedirectory.authorization-clients",
+      "type": "java.util.Map<java.lang.String,com.azure.spring.aad.webapp.AuthorizationClientProperties>",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.allow-telemetry",
+      "type": "java.lang.Boolean",
+      "description": "Telemetry data will be collected if true, or disable data collection.",
+      "defaultValue": true,
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.app-id-uri",
+      "type": "java.lang.String",
+      "description": "App ID URI which might be used in the <code>\"aud\"<\/code> claim of an token.",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.authenticate-additional-parameters",
+      "type": "java.util.Map<java.lang.String,java.lang.Object>",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.authorization-clients",
+      "type": "java.util.Map<java.lang.String,com.azure.spring.autoconfigure.b2c.AuthorizationClientProperties>",
+      "description": "Specify client configuration",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.base-uri",
+      "type": "java.lang.String",
+      "description": "AAD B2C endpoint base uri.",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.client-id",
+      "type": "java.lang.String",
+      "description": "The application ID that registered under b2c tenant.",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
+      }
+    },
+    {
+      "name": "azure.activedirectory.b2c.client-secret",
+      "type": "java.lang.String",
+      "description": "The application secret that registered under b2c tenant.",
+      "deprecation": {
+        "level": "error",
+        "reason": "Todo: add deprecation reason",
+        "replacement": ""
```

Comment:
if there is no replacement for the removed property, then we don't need this element in the json

Reply:
yes

Reply:
yes, for such properties, the replacement property should be removed